apps/test/unit/templates/rubrics/EvidenceLevelsTest.jsx

```diff
@@ -20,8 +21,17 @@ describe('EvidenceLevels', () => {
     expect(wrapper.find('Heading6').props().children).to.equal(
       'Assign a Rubric Score'
     );
-    expect(wrapper.find('Memo(RadioButton)').length).to.equal(2);
-    expect(wrapper.find('BodyThreeText').length).to.equal(2);
+    expect(wrapper.find('Memo(RadioButton)').length).to.equal(
+      DEFAULT_PROPS.evidenceLevels.length
+    );
+    expect(wrapper.find('BodyThreeText').length).to.equal(
+      DEFAULT_PROPS.evidenceLevels.length
+    );
+    const lastEvidenceLevel =
+      DEFAULT_PROPS.evidenceLevels[DEFAULT_PROPS.evidenceLevels.length - 1];
+    expect(wrapper.find('Memo(RadioButton)').at(0).prop('label')).to.equal(
+      UNDERSTANDING_LEVEL_STRINGS[lastEvidenceLevel.understanding]
```

Comment:
This actually tests that the understanding levels are reversed when being displayed.

Reply:
I think this is good -- it relies on the order of the data that's passed in in this test but that's okay.